apps/web/src/app/(dashboard)/dashboard/page.tsx

```diff
@@ -34,22 +34,23 @@ export default async function DashboardPage() {
     }),
   ]);
 
+  const cardData = [
+    { icon: FileCheck, title: 'Completed', status: stats.COMPLETED },
+    { icon: File, title: 'Drafts', status: stats.DRAFT },
+    { icon: Clock, title: 'Pending', status: stats.PENDING },
+  ];
+
   return (
     <div className="mx-auto w-full max-w-screen-xl px-4 md:px-8">
       <h1 className="text-4xl font-semibold">Dashboard</h1>
-
       <div className="mt-8 grid grid-cols-1 gap-4 md:grid-cols-3">
-        <Link href={'/documents?status=COMPLETED'} passHref>
-          <CardMetric icon={FileCheck} title="Completed" value={stats.COMPLETED} />
-        </Link>
-        <Link href={'/documents?status=DRAFT'} passHref>
-          <CardMetric icon={File} title="Drafts" value={stats.DRAFT} />
-        </Link>
-        <Link href={'/documents?status=PENDING'} passHref>
-          <CardMetric icon={Clock} title="Pending" value={stats.PENDING} />
-        </Link>
+        {cardData.map((card) => (
+          <Link key={card.status} href={`/documents?status=${card.status}`} passHref>
```

Comment:
I don't remember why I had `passHref` here tbh

Reply:
I also wasn't sure, as we are not sending the `href` to any child, should I remove that one?